naomi_osullivan/week_01/friday/mta6line.js

```diff
@@ -0,0 +1,88 @@
+var six = ['Astor Place', 'Union Sq6', '23rd6', '28th6', '33rd', 'Grand Central'];
+
+var theSix = function (start, finish, line) {
+  var stopsArray = [];
+//for loop
+if (start < finish) { //w/out positive loop works with neg loop works
+  for (var i = line.indexOf(start); i <= line.indexOf(finish) ; i++) {
+    if(start !== finish ) {
```

Comment:
You should do this test once at the start of your `theSix()` function (and `return` if it's true), rather than in every iteration of the loop, becaue you only need to check for this bad input once, rather than on every iteration of the loop.